releasenotes/notes/fix-proxyconfig-endpoints-json.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
```

Comment:
nit: this is a enhancement

Reply:
Is it? `istioctl proxyconfig all` included endpoints as long as the output wasn't YAML or JSON. That sounds like a bug to me

Reply:
IIRC endpoints summary was added by me, maybe I forgot handling the yaml/json format. It should be a bug.

Reply:
Thanks for the fix